src/runtime-rs/crates/resource/src/cgroups/mod.rs

```diff
@@ -108,17 +108,17 @@ impl CgroupsResource {
     /// delete will move the running processes in the cgroup_manager and
     /// overhead_cgroup_manager to the parent and then delete the cgroups.
     pub async fn delete(&self) -> Result<()> {
-        for cg_pid in self.cgroup_manager.tasks() {
-            self.cgroup_manager.remove_task(cg_pid)?;
+        for cg_pid in self.cgroup_manager.procs() {
```

Comment:
I just wonder if we can remove all tasks when we just remove all pids in `procs` file?